Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:10-alpine
+
+ENV NODE_ENV = "development"
+
+USER node
+
+WORKDIR /home/node
+
+COPY --chown=node:node . .
+
+RUN cp ./api/package*.json ./ && npm install
+
+COPY . .
```

Comment:
This can be removed then since we are copying everything already on line 9 and ensuring node owns the files on copy.

Reply:
Removed those lines.